src/main/resources/entity-types/inventory/simple_locations.json5

```diff
@@ -63,8 +63,8 @@
       queryable: true,
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb ->> 'name'",
-      filterValueGetter: "lower(${tenant_id}_mod_inventory_storage.f_unaccent(:sourceAlias.jsonb ->> 'name'::text))",
-      valueFunction: 'lower(${tenant_id}_mod_inventory_storage.f_unaccent(:value))',
+      filterValueGetter: ":sourceAlias.id::text",
```

Comment:
Um, are you sure this is right for the name?

Reply:
Yes, otherwise we are comparing name to id. In order to make things work we need :(

Reply:
I'm confused... Why would we be comparing an ID to this name column when the name column doesn't have an `idColumnName`?